workflows-scripts/circulation/check-in-check-out/scripts/circ-data-load.sh

```diff
@@ -0,0 +1,59 @@
+# This script needs to be supplied a psql.conf file, which contains the endpoint and credentials of the database.

```

Comment:
Is this script needed only for `check-in`?

Reply:
This script is needed for both check-in and check-out.